### PR TITLE
Change Workflow

### PR DESCRIPTION
**Pull request by piplup55**
Changes workflow files to better support the mod manager pr for ryujinx depends on https://github.com/Ryujinx/Ryujinx/pull/4390

### CODE REVIEW

File: .github/actions/build/action.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ runs:
       cache-from : type=gha,scope=smoo-build-env
       cache-to   : type=gha,scope=smoo-build-env,mode=max
   -
-    name  : Build mod
+    name  : Switch
```
Comment (Istador) [inline .github/actions/build/action.yml]: This step builds the mod for Switch AND for emulators. Naming it `Switch` is conceptually wrong.

**Comment by Istador** [comment]
Please pluralize it everywhere from `Emulator` to `Emulators`, to be consistent in naming to how it was before PR #38.

File: .github/actions/build/action.yml (hunk 1 above)
Comment (piplup55) [reply .github/actions/build/action.yml]: this change was only meant as a test on a local hidden repo, idk how it got here

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -14,12 +14,11 @@ jobs:
...
     strategy:
       matrix:
-        emu : [ Switch, Ryujinx, yuzu ]
+        emu : [ Switch, Emulators ]
     runs-on: ubuntu-latest
     outputs:
       filename1: ${{ steps.set-output.outputs.filename-Switch }}
-      filename2: ${{ steps.set-output.outputs.filename-Ryujinx }}
-      filename3: ${{ steps.set-output.outputs.filename-yuzu }}
+      filename2: ${{ steps.set-output.outputs.filename-Emulator }}
```
Comment (Istador) [inline .github/workflows/release.yml]: Still unpluralized here.

**Comment by Istador** [comment]
I think we can merge this even now without waiting for the mod manager UI.

Using the yuzu build with Ryujinx works when pasting the `SMOO` folder into `%appdata%\Ryujinx\mods\contents\0100000000010000` (right click on the game `Open Mods Directory`).

I only need to adjust the website once this is merged.

**Comment by piplup55** [comment]
> I think we can merge this even now without waiting for the mod manager UI.
> 
> Using the yuzu build with Ryujinx works when pasting the `SMOO` folder into `%appdata%\Ryujinx\mods\contents\0100000000010000` (right click on the game `Open Mods Directory`).
> 
> I only need to adjust the website once this is merged.

yeah you are indeed right no reason to wait then

**Comment by speyejack** [comment]
Checked myself, seemed to work. Merging.

**Pull request merged**